internal/fsutil/create.go

```diff
@@ -121,6 +130,28 @@ func createSymlink(o *CreateOptions) error {
 	return os.Symlink(o.Link, o.Path)
 }
 
+func createHardLink(o *CreateOptions) error {
+	debugf("Creating hard link: %s => %s", o.Path, o.Link)
+	targetInfo, err := os.Lstat(o.Link)
```

Comment:
`targetInfo` is the information for `o.Link` and `linkInfo` is the one from `o.path`, is doesn't make sense. Let's name them `linkInfo` and `pathInfo` in correspondance with `o.Link` and `o.Path`.

Reply:
Fixed in [03ce4d3](https://github.com/letFunny/chisel/pull/17/commits/03ce4d322ccaff4f6ba8c00a32aee039de12c9d3).